src/pages/CovidHistory.js

```diff
@@ -26,6 +33,18 @@ const CovidHistory = ({ covidHistory, setIntakeState, intakeState,setPage, page
   };
 
   const onNext = () => {
+    if (
+      parseInt(moment(intakeState.dateOfDose1Vaccination).year()) >
+        parseInt(moment().year()) ||
+      parseInt(moment(intakeState.dateOfDose2Vaccination).year()) >
+        parseInt(moment().year()) ||
+      parseInt(moment(intakeState.dateOfDose1Vaccination).year()) <
+        parseInt(moment().year()) - 150 ||
+      parseInt(moment(intakeState.dateOfDose2Vaccination).year()) <
+        parseInt(moment().year()) - 150
+    ) {
```

Comment:
what is 150? please move it to constants
```suggestion
    const currentYear = parseInt(moment().year());
    const dontKnowHowToCallYear = currentYear - 150;
    const dose1Year = parseInt(moment(intakeState.dateOfDose1Vaccination).year());
    const dose2Year = parseInt(moment(intakeState.dateOfDose2Vaccination).year());
    if (dose1Year > currentYear
     || dose2Year > currentYear
     || dose1Year < dontKnowHowToCallYear
     || dose2Year < dontKnowHowToCallYear) {
```

Reply:
resolved